tequilapi/endpoints/connection.go

```diff
@@ -87,19 +88,29 @@ func (ce *connectionEndpoint) GetIP(writer http.ResponseWriter, request *http.Re
 // GetStatistics returns statistics about current connection
 func (ce *connectionEndpoint) GetStatistics(writer http.ResponseWriter, request *http.Request, params httprouter.Params) {
 	stats := ce.statsKeeper.Retrieve()
+
+	duration, err := ce.statsKeeper.GetSessionDuration()
+	if err != nil {
+		newErr := fmt.Errorf("unable to retrieve session duration: %v", err.Error())
+		utils.SendError(writer, newErr, http.StatusUnprocessableEntity)
```

Comment:
Unprocessable entity error is meant to use for http request body validation and is not allowed for GET requests, not to report some internal state problems. Maybe duration can be simply 0 if session is not started? Do we really need to handle that "special" case?

Reply:
Simplified to return zero in this case - no custom http status left.